README.rst

```diff
@@ -1,6 +1,15 @@
 MetaIO
 ======
 
+.. image:: https://circleci.com/gh/Kitware/MetaIO.svg?style=shield
```

Comment:
Since the badges are not very descriptive, I suggest to include a table like [this one](https://github.com/NeurodataWithoutBorders/pynwb/blob/dev/README.rst#build-status)

Reply:
@jcfr Thanks for the review.  Adding...